examples/seismic/viscoacoustic/operators.py

```diff
@@ -38,11 +38,15 @@ def src_rec(p, model, geometry, **kwargs):
     return src_term + rec_term
 
 
-def sls_1st_order(model, geometry, p, r=None, **kwargs):
+def SLS_1st_order(model, geometry, p, r=None, **kwargs):
```

Comment:
I think it should stay lowercase? @FabioLuporini @mloubout

Reply:
yeah should stay lower to be consistent

Reply:
okay. I already fixed it.